src/run.rs

```diff
@@ -266,22 +267,63 @@ pub fn clean_up(
     tmp_dir: TempDir,
     worktree_branch: Option<&str>,
     subprocesses: &mut [Child],
+    prompt: bool,
 ) -> Result<()> {
     if !delete_file.exists() {
         log::info!(
             "Project directory preserved at: {}",
             tmp_dir.into_path().display()
         );
-    } else if worktree_branch.is_some() {
-        let mut command = std::process::Command::new("git");
-        command
-            .args(["worktree", "remove"])
-            .arg(&tmp_dir.path())
-            .arg("--force");
-        ensure!(
-            command.status().context("Could not start git")?.success(),
-            "cannot remove working tree"
-        );
+    } else {
+        let delete = if prompt {
+            let delete;
+            println!("Are you sure you want to delete this project? (Y/n)");
+
+            loop {
+                let mut input = String::new();
+
+                match stdin().read_line(&mut input) {
+                    Ok(_n) => match input.trim() {
+                        "" | "Yes" | "yes" | "Y" | "y" => {
+                            delete = true;
+                            break;
+                        }
+                        "No" | "no" | "N" | "n" => {
+                            delete = false;
+                            break;
+                        }
+                        _ => (),
```

Comment:
```
println!("come again");
```

Reply:
Since it's my feature:
```
Are you sure you want to delete this project? (Y/n)
maybe
hmm, `maybe` doesn't looks like `yes` or `no`
yes
```

Reply:
you do you but at least do the grammar correctly lol `doesn't look like`

Reply:
:pray: sorry